base_comment_template/models/base_comment_template.py

```diff
@@ -124,38 +104,44 @@ class BaseCommentTemplate(models.Model):
         help="This comment template will be available on this models. "
         "You can see here only models allowed to set the coment template.",
     )
-
     domain = fields.Char(
         "Filter Domain",
         required=True,
         default="[]",
         help="This comment template will be available only for objects "
         "that satisfy the condition",
     )
-
-    priority = fields.Integer(
-        default=10, copy=False, help="the highest priority = the smallest number",
+    sequence = fields.Integer(
+        required=True, default=1, help="the highest sequence = the smallest number"
```

Comment:
This help message is a bit confusing... I'd leave it as before, with isn't wrong:
```suggestion
        required=True, default=1, help="the highest priority = the smallest number"
```

Reply:
And the proper expression is:
```suggestion
        required=True, default=1, help="the smaller number = the higher priority"
```